src/store/Modules/Auth.ts

```diff
@@ -0,0 +1,30 @@
+import { Module, VuexModule, Mutation, Action } from "vuex-module-decorators"
+import store from "../index"
+
+const storeValue = localStorage.getItem("vuex")
+const AuthInStore = storeValue ? JSON.parse(storeValue).Auth : false
+@Module({
+  name: "Auth",
+  store: store,
+  dynamic: true,
+})
+export default class Auth extends VuexModule {
+  username: string = "Rahul Shaw"
+  email: string = "rshaw@aapnainfotech.com"
+  get user() {
+    return this.username
+  }
+  get mail() {
+    return this.email
+  }
+
+  @Mutation
+  SET_ALIAS_NAME(username: string) {
```

Comment:
Try to maintain the same naming convention already used in the code. (camelCase)

Reply:
@ankitvats1 when you do review comments select request change so that its clear on the PR.

Reply:
Sure @wildone

Reply:
Resolved, It is all in camelCase format